src/modules/profile/pages/profileSettings/components/ProjectNotifications.tsx

```diff
@@ -0,0 +1,123 @@
+import { useMutation, useQuery } from '@apollo/client'
+import { FormControl, FormLabel, HStack, Select, Switch, VStack } from '@chakra-ui/react'
+import { useState } from 'react'
+import { useParams } from 'react-router'
+
+import { MUTATION_UPDATE_USER_NOTIFICATIONS_SETTINGS } from '@/modules/profile/graphql/mutations/userNotificationsMutation'
+import { QUERY_USER_NOTIFICATIONS_SETTINGS } from '@/modules/profile/graphql/queries/userNotificationSettingsQuery'
+import { Body } from '@/shared/components/typography'
+
+type NotificationConfiguration = {
+  id: string
+  name: string
+  value: string
+  type: string
+  options?: string[]
+}
+
+type NotificationSettings = {
+  notificationType: string
+  isEnabled: boolean
+  configurations: NotificationConfiguration[]
+}
+
+export const ProjectNotifications = () => {
+  const { userId } = useParams()
+
+  const [notificationSettings, setNotificationSettings] = useState<NotificationSettings[]>([])
+
+  useQuery(QUERY_USER_NOTIFICATIONS_SETTINGS, {
+    variables: { userId },
+    onCompleted(data) {
+      setNotificationSettings(data?.settingsNotificationsUserGet.notificationSettings || [])
+    },
+  })
+
+  const [updateNotificationSetting] = useMutation(MUTATION_UPDATE_USER_NOTIFICATIONS_SETTINGS)
+
+  const getConfigValue = (type: string, name: string) => {
+    const setting = notificationSettings.find((s) => s.notificationType === type)
+    return setting?.configurations.find((c) => c.name === name)?.value
+  }
+
+  const getConfigId = (type: string, name: string) => {
+    const setting = notificationSettings.find((s) => s.notificationType === type)
+    return setting?.configurations.find((c) => c.name === name)?.id
+  }
```

Comment:
Same as a previous comment for these, if the values for `type` and `name` are predefined, let's use enums. 

Or if it is already a type we get from he graphql.ts. because it is typed in the backen, lets use that.